src/components/ImportPage/index.js

```diff
@@ -289,69 +289,80 @@ export default ({ isDesktop, authConfig, user }) => {
-          open={selectedDialog === "import-csv-json"}
-          onClose={closeDialog}
-          onAddSamples={onAddSamples}
-        />
+        {selectedDialog === "import-from-coco" ? (
+          <ImportFromCOCODialog
+            open={selectedDialog === "import-from-coco"}
+            onClose={closeDialog}
+            // dataset={dataset}
+            // onChangeDataset={onChangeDataset}
+          />
+        ) : selectedDialog === "import-text-snippets" ? (
+          <ImportTextSnippetsDialog
+            open={selectedDialog === "import-text-snippets"}
+            onClose={closeDialog}
+            onAddSamples={onAddSamples}
+          />
+        ) : selectedDialog === "paste-image-urls" ? (
+          <PasteUrlsDialog
+            open={selectedDialog === "paste-image-urls"}
+            onClose={closeDialog}
+            onAddSamples={onAddSamples}
+          />
+        ) : selectedDialog === "import-from-s3" ? (
+          <ImportFromS3Dialog
+            open={selectedDialog === "import-from-s3"}
+            // onChangeFile={onChangeFile}
+            onClose={closeDialog}
+            user={user}
+            onAddSamples={onAddSamples}
+          />
+        ) : selectedDialog === "upload-to-s3" ? (
+          <UploadToS3Dialog
+            open={selectedDialog === "upload-to-s3"}
+            // onChangeFile={onChangeFile}
+            onClose={closeDialog}
+            user={user}
+            onAddSamples={onAddSamples}
+          />
+        ) : selectedDialog === "export-to-cognito-s3" ? (
+          <ExportToCognitoS3Dialog
+            open={selectedDialog === "export-to-cognito-s3"}
+            onClose={closeDialog}
+            onAddSamples={onAddSamples}
+          />
+        ) : selectedDialog === "import-from-cognito-s3" ? (
+          <ImportFromCognitoS3Dialog
+            open={selectedDialog === "import-from-cognito-s3"}
+            onClose={closeDialog}
+            onAddSamples={onAddSamples}
+          />
+        ) : selectedDialog === "google-drive-file-picker" ? (
+          <ImportFromGoogleDriveDialog
+            open={selectedDialog === "google-drive-file-picker"}
+            onClose={closeDialog}
+            onAddSamples={onAddSamples}
+          />
+        ) : selectedDialog === "import-toy-dataset" ? (
+          <ImportToyDataset
+            open={selectedDialog === "import-toy-dataset"}
+            onClose={closeDialog}
+            onAddSamples={onAddSamplesAsToyDataset}
+          />
+        ) : selectedDialog === "youtube-urls" ? (
+          <ImportFromYoutubeUrls
+            open={selectedDialog === "youtube-urls"}
+            onClose={closeDialog}
+            onAddSamples={onAddSamples}
+          />
+        ) : selectedDialog === "import-csv-json" ? (
+          <ImportUDTFileDialog
+            open={selectedDialog === "import-csv-json"}
+            onClose={closeDialog}
+            onAddSamples={onAddSamples}
+          />
+        ) : (
+          <p></p>
+        )}
```

Comment:
this is good

Reply:
one thing you'll notice is that dialog animations for closing won't work now, but this is a small price to pay for the ease of dev